lib/standalone.mjs

```diff
@@ -250,7 +256,12 @@ export default async function standalone({
 				/**
 				 * If we're fixing, save the file with changed code
 				 */
-				if (postcssResult.root && postcssResult.opts && !postcssResult.stylelint.ignored && fix) {
+				if (
+					postcssResult.root &&
+					postcssResult.opts &&
+					!postcssResult.stylelint.ignored &&
+					Boolean(fix)
```

Comment:
[suggestion] Is this conversion to `Boolean(fix)` necessary? If not, can you revert it to reduce the diff?

Reply:
Fixed in this commit.
[refactor: apply suggestion](https://github.com/stylelint/stylelint/pull/8106/commits/7af47cca0e1d865258d4f5e61ff8801198ae627e)